src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs

```diff
@@ -38,5 +41,7 @@
      [react/view {:style {:height 1 :background-color "#e8ebec" :margin-horizontal 16}}]
      [react/touchable-highlight {:on-press #(re-frame/dispatch [:logout])
                                  :style {:margin-top 60}}
-      [react/view
-       [react/text {:style {:color :red}} "Log out"]]]]))
+
+      [react/view {:style styles/logout-row}
+       [react/text {:style {:color colors/red}} "Log out"]
```

Comment:
I think we should add i18n label instead of "Log out" here.

Reply:
oh ok, nice catch @vkjr ! will add that in :+1: